packages/database/index.node.ts

```diff
@@ -88,13 +85,20 @@ export function registerDatabase(instance: FirebaseNamespace) {
     null,
     true
   );
-
-  if (isNodeSdk()) {
-    module.exports = Object.assign({}, namespace, { initStandalone });
-  }
 }
 
-registerDatabase(firebase);
+try {
+  // If @firebase/app is not present, skip registering database.
+  // It could happen when this package is used in firebase-admin which doesn't depend on @firebase/app.
+  // Previously firebase-admin depends on @firebase/app, which causes version conflict on
+  // @firebase/app when used together with the js sdk. More detail:
+  // https://github.com/firebase/firebase-js-sdk/issues/1696#issuecomment-501546596
+  const firebase = require('@firebase/app').default;
+  registerDatabase(firebase);
```

Comment:
Could this swallow a legit error in `registerDatabase()`?  If the require works fine but there is some error thrown in `registerDatabase()` itself?

Reply:
Yeah, updated to only ignore MODULE_NOT_FOUND error